Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM node:15 as development
```

Comment:
Please revert the changes in this file. Let us know if you need any help. 🙂


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable